### ISSUE TITLE
Per NPM blog use package.json instead of fontify.json for configuration metadata

### ISSUE THREAD

**Issue by noderat** [YoussefKababe/fontify#1]
Per the NPM blog post [npm and front end packaging](http://blog.npmjs.org/post/101775448305/npm-and-front-end-packaging) the package.json file should be used for configuration instead of a seperate configuration file.

**Comment by YoussefKababe** [comment]
You're right, I didn't know I can add my own custom fields to the **package.json** file, this would be definitely better. I'll be pushing an update soon. Thanks!
